### PR TITLE
drivers: spi: nrf: Add new SPIS instances

### PR DESCRIPTION
**Pull request by nordic-krch**

### CODE REVIEW

File: modules/hal_nordic/nrfx/nrfx_config.h [hunk 1]
```diff
@@ -409,6 +409,50 @@
 #define NRFX_SPIS3_ENABLED 1
 #endif
```
Comment (anangl) [inline modules/hal_nordic/nrfx/nrfx_config.h:411]: Drop these blank lines, please. We don't have them for other peripherals, like SPIMs or TWIMs.

File: dts/bindings/spi/nordic,nrf-spis.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: Nordic nRF family SPIS (SPI slave with EasyDMA)
 
 compatible: "nordic,nrf-spis"
 
-include: nordic,nrf-spi-common.yaml
+include: ["nordic,nrf-spi-common.yaml", "memory-region.yaml"]
```
Comment (anangl) [inline dts/bindings/spi/nordic,nrf-spis.yaml:8]: Why is this actually added in this PR? It does not seem to be related.

Comment (nordic-krch) [reply dts/bindings/spi/nordic,nrf-spis.yaml]: Moved to another PR.

**Review by anangl — DISMISSED** [review]

**Comment by nordic-krch** [comment]
@anangl can you re-approve? only rebased.
@gmarull can you take a look?

**Review by anangl — DISMISSED** [review]

**Review by gmarull — DISMISSED** [review]
image

**Review by nika-nordic — APPROVED** [review]

**Review by nordicjm — DISMISSED** [review]
Needs converting to hwmv2

**Comment by nordic-krch** [comment]
@nordicjm rebased. @anangl @gmarull can you re-approve? Only rebased.

**Review by anangl — APPROVED** [review]

**Review by gmarull — APPROVED** [review]
